docs/specs/tripgo.swagger.yaml

```diff
@@ -2978,11 +2985,18 @@ definitions:
               type: string
             timezone:
               type: string
+            identifier:
+              type: string
+            boundaries:
```

Comment:
Should we call this `polygon` instead to be symmetric with the `cityPolygons: true` parameter?